redfish-core/lib/power.hpp

```diff
@@ -314,8 +314,7 @@ inline void requestRoutesPower(App& app)
                     "/xyz/openbmc_project/object_mapper",
                     "xyz.openbmc_project.ObjectMapper", "GetSubTreePaths",
                     "/xyz/openbmc_project/inventory", 0,
-                    std::array<const char*, 2>{
-                        "xyz.openbmc_project.Inventory.Item.Board",
+                    std::array<const char*, 1>{
```

Comment:
Wasn't this already done here: https://github.com/ibm-openbmc/bmcweb/pull/85 ?

Reply:
I agree, so I thought git will resolve that but look like it not doing that

Reply:
Please rebase